src/postgraphile/cli.ts

```diff
@@ -565,16 +568,19 @@ function trimNulls(obj: object): object {
   }, {});
 }
 
-const jwtVerifyOptions: jwt.VerifyOptions = trimNulls({
-  algorithms: jwtVerifyAlgorithms,
-  audience: jwtVerifyAudience,
-  clockTolerance: jwtVerifyClockTolerance,
-  jwtId: jwtVerifyId,
-  ignoreExpiration: jwtVerifyIgnoreExpiration,
-  ignoreNotBefore: jwtVerifyIgnoreNotBefore,
-  issuer: jwtVerifyIssuer,
-  subject: jwtVerifySubject,
-});
+const jwtVerifyOptionsActual: jwt.VerifyOptions =
```

Comment:
I think we should warn people that their other settings are being ignored if they're using `jwtVerifyOptions`. Since this is security-critical, I think throwing an error makes sense:

```suggestion
if (
  jwtVerifyOptions &&
  (jwtVerifyAlgorithms ||
    jwtVerifyAudience ||
    jwtVerifyClockTolerance ||
    jwtVerifyId ||
    jwtVerifyIgnoreExpiration ||
    jwtVerifyIgnoreNotBefore ||
    jwtVerifyIssuer ||
    jwtVerifySubject)
) {
  throw new Error(
    'You may not mix `jwtVerifyOptions` with the legacy `jwtVerify*` settings; please only provide `jwtVerifyOptions`.'
  );
}
const jwtVerifyOptionsActual: jwt.VerifyOptions =
```

Reply:
Rather than using `jwtVerifyOptionsActual` here, could you instead rename the old `jwtVerifyOptions` to, e.g., `rawJwtVerifyOptions` - I find it more obvious if the final state is named sensibly.

Reply:
Sorry, I think I miscommunicated.

To make it clearer what I meant, I would like you to change line 430:

https://github.com/graphile/postgraphile/blob/7feea7cc2975a4fd9626e9f3fef7f47f6323569e/src/postgraphile/cli.ts#L430

to use a different name for the destructured property:

```
jwtVerifyOptions: rawJwtVerifyOptions,
```

---

Line 605 

https://github.com/graphile/postgraphile/blob/7feea7cc2975a4fd9626e9f3fef7f47f6323569e/src/postgraphile/cli.ts#L605

would get the reverse treatment:

```
jwtVerifyOptions,
```

---

And replace `jwtVerifyOptionsActual` here with `jwtVerifyOptions`.

---

For me it's more important that the final variable name is correct rather than the initial destructured one (so doing it this way would be consistent with similar patterns in the rest of the codebase).